diagnostics.yml

```diff
@@ -157,6 +157,22 @@ stages:
                 _BuildConfig: Debug
                 _BuildArch: x64
 
+    - template: /eng/build.yml
+      parameters:
+        name: MacOS_cross
+        osGroup: MacOS_cross
+        strategy:
+          matrix:
+            ${{ if in(variables['Build.Reason'], 'PullRequest') }}:
```

Comment:
If we are going to condition this to just be Pull Requests and not produce any bits internal builds, I'd move the if to before the template.

Reply:
This was a temporary fix until I could come up with an internal queue that could run XCode 12 beta